Doc/library/asyncio-task.rst

```diff
@@ -540,6 +540,14 @@ Task functions
 
    .. deprecated:: 3.4.4
 
+.. function:: wrap_future(future, \*, loop=None)
+
+   Wrap a :class:`concurrent.futures.Future` object in a
+   :class:`Future` object.
+
+   When the original :class:`concurrent.futures.Future` is completed
```

Comment:
The wrapping doesn't end, though.  It's that completion of the wrapped future triggers completion of the wrapper.  Which is what I would expect from a wrapper...and I would likewise expect that result on the wrapper will return the result of the wrapped future, so I'm not even sure this sentence is a good idea.